Runtime/Plugins/GLTFSerialization/Schema/GLTFMaterial.cs

```diff
@@ -157,6 +165,9 @@ public static GLTFMaterial Deserialize(GLTFRoot root, JsonReader reader)
 					case "doubleSided":
 						material.DoubleSided = reader.ReadAsBoolean().Value;
 						break;
+					case "shaderName":
+						material.OriginalUnityShaderName = reader.ReadAsString();
+						break;
```

Comment:
This may violate the GLTF specification.

Please test a gltf exported with this code with https://github.khronos.org/glTF-Validator/

Reply:
If it does violate the spec, you may be able to use the extras field for this data:
https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html